### ISSUE TITLE
SDA to do list

### ISSUE THREAD

**Issue by liyu711** [FlintHill/SUAS-Competition#64]
-Integration
-Unit testing for overlapping obstacles
-Beginning sim testing
-Moving obstacles

**Comment by BrightStraightReality** [comment]
Updated to do for 10/22/17:

Thoroughly test SDA Package, SDA converter

**Comment by DarkmatterVale** [comment]
Here is the current to do list:

- [x] Stationary Obstacles
    - [x] Complete interop testing (11/10)
    - [x] Successfully avoid demo obstacle test on Spirit IX.1 (11/13)
    - [ ] Design full "test suite" in interop, and successfully avoid all obstacles --> Completion of this marks the end of stationary obstacle development (11/17)
- [ ] Moving Obstacles
    - [ ] Logic completed (11/10)
    - [ ] Development sprint for a basic version, with unit tests completed (11/17)

**Comment by DarkmatterVale** [comment]
Ok so we are a little behind, but we are making progress. The last issue before we can close this is to get the full test for the obstacles working. Lets try to get this done by the first week after we get back from break.

**Comment by DarkmatterVale** [comment]
I'm closing this out because it is no longer relevant.
